examples/src/main/java/ImageClassification.java

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright 2021 YANDEX LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ai.toloka.client.v1.LangIso639;
+import ai.toloka.client.v1.ModificationResult;
+import ai.toloka.client.v1.aggregatedsolutions.AggregatedSolutionSearchRequest;
+import ai.toloka.client.v1.aggregatedsolutions.DawidSkenePoolAggregatedSolutionRequest;
+import ai.toloka.client.v1.impl.TolokaClientFactoryImpl;
+import ai.toloka.client.v1.impl.transport.DefaultHttpClientConfiguration;
+import ai.toloka.client.v1.pool.MixerConfig;
+import ai.toloka.client.v1.pool.Pool;
+import ai.toloka.client.v1.pool.PoolDefaults;
+import ai.toloka.client.v1.pool.PoolStatus;
+import ai.toloka.client.v1.pool.filter.ArrayInclusionOperator;
+import ai.toloka.client.v1.pool.filter.CompareOperator;
+import ai.toloka.client.v1.pool.filter.Connective;
+import ai.toloka.client.v1.pool.filter.Expression;
+import ai.toloka.client.v1.pool.qualitycontrol.CollectorConfig;
+import ai.toloka.client.v1.pool.qualitycontrol.QualityControl;
+import ai.toloka.client.v1.pool.qualitycontrol.QualityControlConfig;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleAction;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleCondition;
+import ai.toloka.client.v1.pool.qualitycontrol.RuleConfig;
+import ai.toloka.client.v1.project.AssignmentsIssuingType;
+import ai.toloka.client.v1.project.Project;
+import ai.toloka.client.v1.project.spec.FieldSpec;
+import ai.toloka.client.v1.project.spec.TaskSpec;
+import ai.toloka.client.v1.project.spec.TbTaskViewSpec;
+import ai.toloka.client.v1.task.KnownSolution;
+import ai.toloka.client.v1.task.Task;
+import ai.toloka.client.v1.task.TaskCreateRequestParameters;
+import ai.toloka.client.v1.task.TaskSearchRequest;
+import ai.toloka.client.v1.userrestriction.DurationUnit;
+import ai.toloka.client.v1.userrestriction.UserRestrictionScope;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+
+/**
+ * Example of using toloka-java-sdk for image classifications.
+ * To find details read examples/image_classification/README.md.
```

Comment:
you can provide full url link to md file

Reply:
Done